js/core/net/tcp-timer.js

```diff
@@ -13,27 +13,19 @@
 // limitations under the License.
 
 'use strict';
-var tcpSocketState = require('./tcp-socket-state');
-var connections = new Set();
+// const tcpSocketState = require('./tcp-socket-state');
+const connections = new Set();
 
-function timeoutHandler() {
-  for (var connSocket of connections) {
-    connSocket._timerTick();
-  }
+let initTimeout;
```

Comment:
ahh, interesting, I guess cases like that are pretty rare. I think either of those is fine:
- disable it for function declarations. http://eslint.org/docs/rules/no-use-before-define
- rewrite it using `setTimeout()` twice 

(not using `setInterval` here to prevent event queue overflow in case system can't process events fast enough)